cmd/root.go

```diff
@@ -200,18 +200,12 @@ func initConfig() {
 		}
 	}
 
-	xdgHome, found := os.LookupEnv("XDG_CONFIG_HOME")
-	if found {
-		gConfigFolder = path.Join(xdgHome, "exoscale")
-	} else {
-		home, found := os.LookupEnv("HOME")
-		if found {
-			gConfigFolder = path.Join(home, ".config", "exoscale")
-		} else {
-			// The XDG spec specifies a default XDG_CONFIG_HOME in $HOME/.config
-			gConfigFolder = path.Join(usr.HomeDir, ".config", "exoscale")
-		}
+	cfgdir, err := os.UserConfigDir()
+	if err != nil {
+		log.Fatal(fmt.Errorf("could not find configuration directory: %s", err))
+		return
```

Comment:
```suggestion
```
[`log.Fatal`](https://godoc.org/log#Logger.Fatal) calls calls `os.Exit(1)`, no need to return